src/mentions/plugin.js

```diff
@@ -6,8 +6,8 @@
 import { Plugin, PluginKey } from 'prosemirror-state';
 import { Decoration, DecorationSet } from 'prosemirror-view';
 
-export const triggerCharacters = char => $position => {
-  const regexp = new RegExp(`(?:^)?${char}[^\\s${char}]*`, 'g');
+export const triggerCharacters = (char, minChars = 0) => $position => {
```

Comment:
@iamsivin I would suggest to add some comments for this changes.

Reply:
@muhsin-k Done